test/integration/test_integration.py

```diff
@@ -1487,5 +1487,39 @@ def test_get_model_size_aqt(self, api, test_device, test_dtype):
 
 
 
+class TestBenchmarkModel(unittest.TestCase):
+    
+    class ToyLinearModel(torch.nn.Module):
+        def __init__(self, m=64, n=32, k=64):
+            super().__init__()
+            self.linear1 = torch.nn.Linear(m, n, bias=False)
+            self.linear2 = torch.nn.Linear(n, k, bias=False)
+
+        def example_inputs(self, batch_size=1, dtype=torch.float32, device="cpu"):
+            return (torch.randn(batch_size, self.linear1.in_features, dtype=dtype, device=device),)
+
+        def forward(self, x):
+            x = self.linear1(x)
+            x = self.linear2(x)
+            return x
+
+    def run_benchmark_model(self, device):
+        # params
+        dtype = torch.bfloat16
+        m = self.ToyLinearModel(1024, 1024, 1024).eval().to(dtype).to(device)
+        m_bf16 = copy.deepcopy(m)
+        example_inputs = m.example_inputs(dtype=dtype, device=device)
+        m_bf16 = torch.compile(m_bf16, mode='max-autotune')
```

Comment:
Number of runs should be closer to 1 and we might not need the max auto tune mode. The idea is here is this test should be as fast as possible so we just know that the benchmark function isn't broken. Intent is not to run a benchmark